network-api/networkapi/templates/fragments/environment_prefix.html

```diff
@@ -0,0 +1 @@
+{% if APP_ENVIRONMENT == "Staging" %} [S] {% elif APP_ENVIRONMENT == "Review" %} [RA] {% endif %}
```

Comment:
with apologies, but it feels like this is probably better done as "not a context_processor" and instead using a tag with a simple `def environment_prefix(context): return settings.APP_ENVIRONMENT` and then using `{% environment_prefix %}` everywhere instead. It will be both less code in terms of "new code" as well as less code in terms of "things we might typo when we add it elsewhere/update it".

Reply:
Hi Pomax, no apology necessary, I am all for keeping the code base as easy to maintain as possible 😄 thank you for the review and great suggestion. I have updated the PR to now use a tag for the env prefix instead!